tests/xliff/xliff_test.py

```diff
@@ -0,0 +1,183 @@
+import filecmp
+import io
+import re
+import zipfile
+
+import pytest
+
+from django.urls import reverse
+
+from integreat_cms.cms.models import Page
+
+from ..conftest import ANONYMOUS, STAFF_ROLES, PRIV_STAFF_ROLES, MANAGEMENT, EDITOR
+
+
+@pytest.mark.django_db
+def test_xliff_export(login_role_user, settings, tmp_path):
+    """
+    This test checks whether the xliff export works as expected
+
+    :param login_role_user: The fixture providing the http client and the current role (see :meth:`~tests.conftest.login_role_user`)
+    :type login_role_user: tuple
+
+    :param settings: The fixture providing the django settings
+    :type settings: :fixture:`settings`
+
+    :param tmp_path: The fixture providing the directory for temporary files for this test case
+    :type tmp_path: :fixture:`tmp_path`
+    """
+    # Set debug mode so xliff files are served
+    settings.DEBUG = True
+    client, role = login_role_user
+    export_xliff = reverse(
+        "download_xliff", kwargs={"region_slug": "augsburg", "language_slug": "en"}
+    )
+    response = client.post(export_xliff, data={"selected_ids[]": [1, 2]})
+    print(response.headers)
+    if role in STAFF_ROLES + [MANAGEMENT, EDITOR]:
+        # If the role should be allowed to access the view, we expect a successful result
+        assert response.status_code == 302
+        page_tree = reverse(
+            "pages", kwargs={"region_slug": "augsburg", "language_slug": "en"}
+        )
+        assert response.headers.get("Location") == page_tree
+        response = client.get(page_tree)
+        print(response.headers)
+        # The download link to the xliff file is contained in the message
+        match = re.search(
```

Comment:
I'd prefer a real XML parser in such cases, but I do not object a regex for this limited scope.

Reply:
Good idea, I replaced the regex with:
```
parsed_content = fromstring(response.content.decode("utf-8"))
download_links = [
    link.attrib["href"]
    for link in parsed_content.iter("a")
    if "data-auto-download" in link.attrib
]
```